internal/conductorr/api/api.go

```diff
@@ -65,6 +67,11 @@ var whitelistPaths = []string{
 	"/api/v1/checkAuth",
 	"/api/csl.wasm",
 	"/api/v1/logout",
+
+	// *arr interop
+	"/api/v1/interop/radarr/api/v3/*",
+	"/api/v1/interop/radarr/api/v3/movie",
+	"/api/v1/interop/radarr/api/v3/movie/*",
```

Comment:
there are probably more intelligent ways of doing this but this just makes it easy to use `path.Match`

Reply:
So these endpoints don't need to be authenticated?